src/nwb2bids/_converters/_session_converter.py

```diff
@@ -55,7 +56,16 @@ def from_nwb_paths(
             if nwb_path.is_file():
                 all_nwbfile_paths.append(nwb_path)
             elif nwb_path.is_dir():
-                all_nwbfile_paths += list(nwb_path.rglob(pattern="*.nwb"))
+                all_nwbfile_paths += [
+                    path
+                    for path in nwb_path.rglob(pattern="*.nwb")
+                    if (
+                        # Ignore contents in hidden folders; e.g. .git, __pycache__, etc.
```

Comment:
```suggestion
                        # Ignore contents in hidden folders; e.g. .git since contains .gi/annex which might include file extensions, and that MacOS trashy folder, __pycache__, etc.
```